src/js/script.js

```diff
@@ -26,6 +26,31 @@ function miniclock(secs) {
     sec
   )}`;
 }
+//modify from https://stackoverflow.com/questions/36856232/write-add-data-in-json-file-using-node-js/36857101
+function writeToFile(jsonpath,taskDate,taskName,taskStart,workLength,breakLength,allPeriods){
+  const fs = window.require('fs');
+  const path = window.require('path');
+  console.log("you are here");
+  var jPath = path.join(__dirname,'..','json', jsonpath);
+  console.log(jPath);
+  fs.readFile(jPath, 'utf8', function readFileCallback(err, data){
+    if (err){
+        console.log(err);
```

Comment:
```suggestion
        throw err;
```
Let's just throw the error when we see it, rather than silently printing it out.